GOVERNANCE.md

```diff
@@ -57,7 +57,18 @@ If the PR author is a maintainer or an owner of given component, this counts as
 
 Proposals can be opened against the [proposals repository](https://github.com/strimzi/proposals).
 Proposals should cover any changes to the Strimzi project which might significantly impact its users or the project direction.
-Proposals are approved by a ⅔ majority maintainers vote.
+
+Voting about proposals is using +1, 0, -1 votes and their fractions, where:
+* +1 means "yes"
+* -1 means "no"
+* The numbers between +1 and -1 indicate how strongly you feel about the proposal
+
+_(Inspired by [Apache Software Foundation voting](https://www.apache.org/foundation/voting.html#expressing-votes-1-0-1-and-fractions))_
+
+Proposals are approved when they receive at least three +1 votes from maintainers and no -1 votes from maintainers.
```

Comment:
We should add that votes from non-maintainers are also welcome, but they just won't count towards approval.